app/modules/screen-sharing/controllers/AuthenticationCtrl.js

```diff
@@ -28,6 +28,15 @@ App.controllers.authenticationCtrl = (function ($, App) {
             desc = null;
         }
 
+        function clickExtension(e){
+            e.preventDefault();
+            chrome.webstore.install('https://chrome.google.com/webstore/detail/jlpojfookfonjolaeofpibngfpnnflne', function(){
+                console.log('Successfully installed Chrome Extension');
```

Comment:
that's a good point.. I will also make an issue in Jira so that we can "detect extensions after the respoke client has been loaded"